tests/features/licenseexport_spdx.feature

```diff
@@ -36,41 +36,78 @@ Scenario: User Downloads license information for SPDX SBOM from SBOM Explorer pa
 Scenario: Verify the files on downloaded SPDX SBOM license ZIP
 	Given User has Downloaded the License information for SPDX SBOM
 	When User extracts the Downloaded license ZIP file
-	Then Extracted files should contain two CSVs, one for Package License combination and another one for License reference
+	Then Extracted files should contain two CSVs, one for Package license information and another one for License reference
 
 Scenario: Verify the headers on SPDX SBOM package License CSV file
 	Given User extracted the SPDX SBOM license compressed file
-	When User Opens the package license combination file
-	Then The file should have the following headers - name, namespace, group, version, package reference, license, license name and alternate package reference
+	When User Opens the package license information file
+	Then The file should have the following headers - name, namespace, group, version, package reference, license, license name, license expression and alternate package reference
 
 Scenario: Verify the headers on SPDX SBOM License reference CSV file
 	Given User extracted the SPDX SBOM license compressed file
 	When User Opens the license reference file
 	Then The file should have the following headers - licenseId, name, extracted text and comment
 
 Scenario: Verify the license information for a package with single license
-	Given User is on package license combination file
+	# Test data https://drive.google.com/drive/folders/1Z6y6gMegutBeUuc_8LkpYxKeGz_KYG9H?usp=drive_link
```

Comment:
@mrrajan scenarios that do not say if they relate to CycloneDX or SPDX SBOMs are general scenarios (no matter which format is used)?

Reply:
The feature file is for SPDX.